src/index.ts

```diff
@@ -103,3 +103,5 @@ export default function swcPreserveDirectivePlugin(): Plugin {
     }
   }
 }
+
+export default swcPreserveDirective
```

Comment:
I think we can create a named export with a shorter name just other rollup plugins
```suggestion
export default swcPreserveDirective
export { swcPreserveDirective as preserveDirective }
```

Reply:
Done in [`adee1ff` (#5)](https://github.com/huozhi/rollup-plugin-swc-preserve-directives/pull/5/commits/adee1fffdd1bf2ea185d5f583eddc04d5f7afd30).